### PR TITLE
expose RESULT_UNKNOWN error messages

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_worker/plugins/evaluate.py [hunk 1]
```diff
@@ -115,7 +119,7 @@ def _get_result_stub(status: int = RESULT_UNKNOWN,
                          hysteresis: int = RESULT_OK):
         return {
             'status': status,
-            'message': message,
+            'message': message[:200],
```
Comment (deejay1) [inline alamo_worker/plugins/evaluate.py]: why that truncate?

Comment (ch40s3k) [reply alamo_worker/plugins/evaluate.py]: https://github.com/RulersOfAsgard/alamo-common/blob/develop/alamo_common/parser/__init__.py#L175

File: alamo_worker/plugins/druid.py [hunk 2]
```diff
@@ -145,17 +145,23 @@ def supported_types(self):
...
             )
 
         # Try/Except block because response can be Dict or List
         try:
             # http://druid.io/docs/latest/querying/querying.html
             # Query Errors section
             if response.get('error'):
+
+                message = response.get('error', '')
+                # `errorMessage` may be null
```
Comment (pnuckowski) [inline alamo_worker/plugins/druid.py]: message = response.get('error') or response.get('errorMessage', '')

File: alamo_worker/plugins/evaluate.py (hunk 1 above)
Comment (pnuckowski) [reply alamo_worker/plugins/evaluate.py]: pass original message

File: alamo_worker/plugins/druid.py (hunk 2 above)
Comment (ch40s3k) [reply alamo_worker/plugins/druid.py]: * message = response.get('errorMessage', '') or response.get('error')

Comment (pnuckowski) [reply alamo_worker/plugins/druid.py]: or maybe 
`message = json.dumps(response)`?

Comment (ch40s3k) [reply alamo_worker/plugins/druid.py]: No, too many information.

File: alamo_worker/plugins/evaluate.py (hunk 1 above)
Comment (ch40s3k) [reply alamo_worker/plugins/evaluate.py]: ?

Comment (ch40s3k) [reply alamo_worker/plugins/evaluate.py]: So after consultation it seems to be intentional :)

Comment (pnuckowski) [reply alamo_worker/plugins/evaluate.py]: Discussed f2f.
Error messages are and should be passed in original size (no trim).

**Pull request merged**